spec/fixtures/hieradata/pe.yaml

```diff
@@ -0,0 +1,8 @@
+---
+simp_options::puppet::server: "%{facts.fqdn}"
+simp_options::puppet::ca: "%{facts.fqdn}"
+simp_options::puppet::server_distribution: 'PE'
+pupmod::puppet_server: '1.2.3.4'
+apache::rsync_server: '127.0.0.1'
+apache::ssl::client_nets: '127.0.0.1/32'
```

Comment:
AFAICT, this can be slimmed down to only the `simp_options::puppet::server_distribution` key/value.